var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -15,6 +15,8 @@ class Libfabric(AutotoolsPackage):
     git      = "https://github.com/ofiwg/libfabric.git"
 
     version('develop', branch='master')
```

Comment:
You can actually rename this version to `master` if you want now that #1983 has been merged.